forms/src/main/java/org/odk/collect/forms/FormsRepository.java

```diff
@@ -19,6 +19,9 @@
     @Nullable
     Form getOneByMd5Hash(@NotNull String hash);
 
+    @Nullable
+    Form getOneNotDeletedByMd5Hash(@NotNull String hash);
```

Comment:
I think we could just check the `Form#isDeleted` on the returned value of `getOneByMd5Hash` instead of introducing this function. That would mean we aren't increasing the surface area of the `FormsRepository` (and the number of tests required). What do you think?

Reply:
Yeah we can do that like that too.